apps/jira-node/src/services/monday-service.js

```diff
@@ -0,0 +1,108 @@
+const fetch = require('node-fetch');
+const initMondayClient = require('monday-sdk-js');
+
+const triggerMondayIntegration = async (webhookUrl, data = {}) => {
+  fetch(webhookUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: process.env.MONDAY_SIGNING_SECRET,
+    },
+    body: JSON.stringify({
+      trigger: {
+        outputFields: {
+          ...data,
+        },
+      },
+    }),
+  });
+};
+
+const queryColumns = async (token, boardId) => {
+  try {
+    const mondayClient = initMondayClient({ token });
+    const query = `query queryBoards($boardId: [Int]) { boards(ids: $boardId) { columns { id } } }`;
+    const variables = { boardId };
+    var response = await mondayClient.api(query, { variables });
+    response = response.data.boards[0].columns;
+    const columnArray = []; 
+    response.forEach(column => {
```

Comment:
why do we need it? we are actually just cloning one array to another, why?

if we need to shallow clone, can be done like this
const columnArray = [..response]

if we need deep clone, can be done like this:
const columnArray = JSON.parse(JSON.stringify(response))


But I think in this case we can just return response